.github/workflows/compliance.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       run: |
         pip3 install setuptools
         pip3 install wheel
-        pip3 install python-magic lxml junitparser gitlint pylint pykwalify yamllint clang-format unidiff
+        pip3 install python-magic lxml junitparser gitlint pylint pykwalify yamllint clang-format unidiff sphinx-lint
```

Comment:
I do wonder why we can't use `scripts/requirements-compliance.txt` here?

Reply:
It should be cloned at this point, not too sure either.